modules/oe_webtools_globan/config/schema/oe_webtools_globan.schema.yml

```diff
@@ -0,0 +1,20 @@
+oe_webtools_globan.settings:
+  type: config_object
+  label: 'Webtools Globan settings'
+  mapping:
+    display_eu_flag:
+      type: boolean
+      label: 'Display EU flag'
+      description: 'Enable or disable the EU flag icon in the Global Banner.'
+    background_theme:
```

Comment:
Can we change this to something that makes more sense? Like to store a string `light` or `dark`? It's not really a boolean if we are choosing between two values that are not really on/off, yes/no, etc. Especially if the config name is simply called "background_theme"